region/norfair/east/Double Chamber.json

```diff
@@ -723,6 +1016,55 @@
       ],
       "devNote": ["The Kamers make this tricky since it's easy to bonk on them."]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"or": [
+          "SpaceJump",
+          "canWalljump",
+          {"and": [
+            "HiJump",
+            "SpeedBooster"
+          ]},
+          "canTrickyDashJump",
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 28,
+              "gentleUpTiles": 3,
+              "gentleDownTiles": 3,
+              "openEnd": 0
+            }},
+            "canMidairShinespark",
```

Comment:
I don't know where we are jumping to or sparking.  Contextually you pass through a descending platform.  But it seems like canShinechargeMovementComplex + a description.

Reply:
This is just sparking up, similar to strat 13. It's not super likely to be useful until CF options are added, but I pointed out the speed options, since they are a bit more obscure. There's also the energy free shinespark option which could make it applicable.

Reply:
It's because there is a refill which isn't modeled and then heat damage (+ shinespark damage) taken while reaching the door. This strat is just saying all of those cancel out, partially to make it so that this strat won't be used to gain energy. Without modeling the refill I dont see any other way to model the spark, though we could change it from 1/1 frames/excess frames to 0/0 or 17/17; not sure which of the three would be most clear (also not sure if 0 would validate).